diff --git a/recovery.fstab b/recovery.fstab
index 0e0854a..03b170d 100644
--- a/recovery.fstab
+++ b/recovery.fstab
@@ -13,6 +13,8 @@
 /system			ext4	/dev/block/bootdevice/by-name/system
 /system_root			ext4	/dev/block/bootdevice/by-name/system flags=fsflags=ro
 /system_image	emmc	/dev/block/bootdevice/by-name/system
+/vendor			ext4	/dev/block/bootdevice/by-name/vendor flags=backup=1;display="Vendor"
+/vendor_image	emmc	/dev/block/bootdevice/by-name/vendor      flags=backup=1;display="Vendor"
 /efs1			emmc	/dev/block/bootdevice/by-name/modemst1						flags=backup=1;display=EFS
 /efs2			emmc	/dev/block/bootdevice/by-name/modemst2						flags=backup=1;subpartitionof=/efs1
 /efsg           emmc	/dev/block/bootdevice/by-name/fsg                           flags=backup=1;subpartitionof=/efs1
